autoupdate-docs-travis.sh

```diff
@@ -8,16 +8,6 @@ set -e
 
```

Comment:
We can rename this to just autoupdate-docs.sh and change the comment at the top since it's not on travis anymore.